wallets/README.md

```diff
@@ -0,0 +1,86 @@
+# Wallets
+
+## Wallet object
+
+```
+{
+  userId: <userId>,
+  isActive: bool,
+  currencies: {
+    neelam: number,
+    dinero: number
+  }
+}
+```
+
+## **Requests**
+
+|                     Route                     |                  Description                   |
+| :-------------------------------------------: | :--------------------------------------------: |
+|          [GET /wallet](#get-wallet)           |        Returns  the user wallet details        |
+| [GET /wallet/:username](#get-walletusername) | Returns the user wallet details for a username |
+
+
+## **GET /wallet**
+
+Returns  the user wallet details
+
+- **Params**  
+  None
+- **Query**  
+  None
+- **Body**  
+  None
+- **Headers**  
+  Content-Type: application/json
+- **Cookie**  
+  rds-session: `<JWT>`
+- **Success Response:**
+- **Code:** 200
+  - **Content:**
+
+```
+{
+  message: 'Wallets returned successfully!'
+  wallet: {
+    id:<walletId>,
+    userId: <userId>,
+    isActive: bool,
+    currencies: {
+      neelam: number,
+      dinero: number
+    }
+  }
+}
+```
+
+- **Error Response:**
+  - **Code:** 401
+    - **Content:** `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
+  - **Code:** 500
+    - **Content:** `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'An internal server error occurred' }`
+
+## **GET wallet/username**
+
+Returns the user wallet details for a username.
+
+- **Params**  
+  None
+- **Query**
+  None
+- **Body**  
+  None
+- **Headers**  
+  Content-Type: application/json
+- **Cookie**  
+  rds-session: `<JWT>`
```

Comment:
As per member's page requirement, user can see the wallet details of all the members right.
So with the cookie attached as header here, it won't be possible for new user as user would not be having token.
@Vividh25 Could you add more details here please and let me know If I am thinking wrong here

Reply:
Yeah, this route has to be public

Reply:
Ye you are right @harshith-venkatesh, this route needs to be public in order to make the member page feature work.